pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -336,10 +336,6 @@ stdenv.mkDerivation ({
 
     echo configureFlags: $configureFlags
     ${setupCommand} configure $configureFlags 2>&1 | ${coreutils}/bin/tee "$NIX_BUILD_TOP/cabal-configure.log"
```

Comment:
I think we can drop the `tee` here too... Are we looking at cabal-configure.log anywhere else?